common/djangoapps/entitlements/models.py

```diff
@@ -221,12 +221,12 @@ def to_dict(self):
             'expired_at': self.expired_at
         }
 
-    @classmethod
     def set_enrollment(cls, entitlement, enrollment):
```

Comment:
This should be (self, enrollment), right?

Reply:
This was a merge conflict. I have cleaned this up.